pytrustnfe/test/test_nfse_paulistana_endereco_tomador.py

```diff
@@ -0,0 +1,55 @@
+# coding=utf-8
+
+import unittest
+from pytrustnfe.xml import render_xml, sanitize_response
+from const import NFSE, DEFAULT_RPS
+
+
+attrs = ['TipoLogradouro', 'Logradouro', 'NumeroEndereco', 'ComplementoEndereco', 'Bairro', 'CEP']
+
+template_path = '../nfse/paulistana/templates'
```

Comment:
I think that a better idiom would be something like
```python
template_path = os.path.join(os.path.dirname(__file__), '..', 'nfse', 'paulistana', 'templates')
```
The reason is that it would make possible to use it in multiple environments, no only posix ones.

Reply:
I agree! I'll change to this :smile: